stock_location_empty/models/stock_location.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class StockLocation(models.Model):
+    _inherit = 'stock.location'
+
+    stock_amount = fields.Integer(
+        compute='_compute_location_amount', search='_search_location_amount'
+    )
+
+    @api.multi
+    def _search_location_amount(self, operator, value):
+        if operator not in ('=', '!=', '<', '<=', '>', '>='):
+            return []
+        # pylint: disable=sql-injection
```

Comment:
Don't do that. Pylint error says that you have to pass parameters to second argument:

`cr.execute("blabla %s", value)`